cmd/vaults/config.go

```diff
@@ -95,96 +60,15 @@ func defaultConfigLocation(dir string) (string, error) {
 			return "", fmt.Errorf("home dir: %s", err)
 		}
 
-		dir = path.Join(dir, ".basin")
+		dir = path.Join(dir, ".vaults")
 	}
 
-	_, err := os.Stat(dir)
```

Comment:
simplifies to mkdir logic. now it does not check if dir exists, it simply tries to create and ignore error if it already exists